.github/workflows/pull-request.yml

```diff
@@ -8,7 +8,8 @@ on:
       - opened
       - labeled
       - unlabeled
-
+permissions:
+  pull-requests: write
```

Comment:
I think this belongs on the job, or rather, is where it would be better served!

```
jobs:
  check_labels:
    name: Check PR Labels
    runs-on: ubuntu-latest
    permissions:
        pull-requests: write
```